func/HeroicBashLauncher.py

```diff
@@ -42,9 +40,6 @@
         frombash(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
     else:
         launchflatpakgame(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4], sys.argv[5])
-elif checkzenity != 0:
-    
-    print("Zenity not installed. Please consider doing so and try again.")
 else:
+    zenity_popup(type=error, title="Process Stopped", text="Looks like you have not installed Heroic Games Launcher or installed any game\n\nPlease consider doing so and try again")
```

Comment:
I believe it should be `type = "error"`. The same applies to in artwork.py, checkparameters.py and steam.py.

Reply:
Yup, bad find & replace on my part :P